ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -674,7 +674,17 @@ struct FrameworkBuilder {
       let binaryName = frameworkPath.lastPathComponent.replacingOccurrences(of: ".framework",
                                                                             with: "")
       let fatBinary = frameworkPath.appendingPathComponent(binaryName).resolvingSymlinksInPath()
-      let infoPlist = frameworkPath.appendingPathComponent("Info.plist").resolvingSymlinksInPath()
+      let plistPathComponents = {
+        if platform == .catalyst || platform == .macOS {
```

Comment:
Run style after verifying the zip builds

Reply:
yep, have that and changelog done locally. will push now that zip is done. Still working on verifying when using all artifacts from the zip. Encountering some 401 network errors at the moment.